products/keda.md

```diff
@@ -0,0 +1,72 @@
+---
+title: KEDA
+category: server-app
+tags: cncf
+permalink: /keda
+releasePolicyLink: https://keda.sh/docs/latest/operate/cluster/
```

Comment:
```suggestion
releasePolicyLink: https://github.com/kedacore/governance/
```

This link looks more appropriate as it contains more information, WDYT ?

Reply:
IMO the most important thing for a release / support policy is to be publicly documented. So I think even if it does not match the reality I think we should go ahead with https://github.com/kedacore/keda/blob/main/SECURITY.md:

> KEDA commits to supporting the n-2 version minor version of the current major release; as well as the last minor version of the previous major release.

WDYT ?